scss/grid/_theme.scss

```diff
@@ -122,6 +122,10 @@
         }
     }
 
+    col.k-sorted,
+    th.k-sorted {
+        background-color: rgba( contrast( $panel-bg ), .02 );
```

Comment:
it's an important variable, so it will be best if extract it as `$grid-sorted-bg`